pageindicatorview/src/main/java/com/rd/PageIndicatorView.java

```diff
@@ -157,16 +157,6 @@ public void onPageSelected(int position) {
     public void onPageScrollStateChanged(int state) {/*empty*/}
 
     /**
-     * Set number of circle indicators to be displayed.
```

Comment:
We cannot remove methods completely in library that is already in release, cuz some users could have compatibility issues because of it.